modules/runtime/pages/bonita-docker-installation.adoc

```diff
@@ -411,6 +411,38 @@ docker run --name bonita -h <hostname> -v ~/bonita-lic/:/opt/bonita_lic/ \
 --
 ====
 
+== Adding internal CA certificates to the truststore
```

Comment:
It's recommanded to set a section archor with `[#my-id]` rather than the generated one